### PR TITLE
Fix w3dview camera movement

### PR DESCRIPTION
**Pull request by xezon**
Left and right camera movement was previously inverted. This fixes it.

### CODE REVIEW

File: src/tools/w3dviewer/graphicview.cpp [hunk 1]
```diff
@@ -236,7 +236,7 @@ void CGraphicView::OnMouseMove(UINT nFlags, CPoint point)
 
         v.X = (((float)point.x - (float)(r.right >> 1)) / (float)(r.right >> 1)
                   - ((float)m_mousePos.x - (float)(r.right >> 1)) / (float)(r.right >> 1))
-            * m_radius * -1.0f;
+            * m_radius;
```
Comment (jonwil) [inline src/tools/w3dviewer/graphicview.cpp]: The * -1.0f does exist in the original binary.

File: src/tools/w3dviewer/graphicview.cpp [hunk 2]
```diff
@@ -236,7 +236,7 @@ void CGraphicView::OnMouseMove(UINT nFlags, CPoint point)
 
         v.X = (((float)point.x - (float)(r.right >> 1)) / (float)(r.right >> 1)
                   - ((float)m_mousePos.x - (float)(r.right >> 1)) / (float)(r.right >> 1))
-            * m_radius * -1.0f;
+            * m_radius;
 
         v.Y = (((float)point.y - (float)(r.bottom >> 1)) / (float)(r.bottom >> 1)
                   - ((float)m_mousePos.y - (float)(r.bottom >> 1)) / (float)(r.bottom >> 1))
```
Comment (jonwil) [inline src/tools/w3dviewer/graphicview.cpp]: On this line, the expressions are ((float)(r.bottom >> 1) - (float)point.y) and ((float)(r.bottom >> 1) - (float)m_mousePos.y)

File: src/tools/w3dviewer/graphicview.cpp (hunk 1 above)
Comment (xezon) [reply src/tools/w3dviewer/graphicview.cpp]: I can absolutely confirm that this `* -1.0f` causes inverted movement (camera moves left when moving right).

Comment (jonwil) [reply src/tools/w3dviewer/graphicview.cpp]: I tested the latest w3dview from the Thyme trunk alongside the original Westwood w3dview and when I hold down both mouse buttons and move left and right it behaves the same. But if I use the w3dview from this PR, it behaves differently.
That means the left and right movement currently in Thyme is correct to the original as far as I can tell.

File: src/tools/w3dviewer/graphicview.cpp (hunk 2 above)
Comment (jonwil) [reply src/tools/w3dviewer/graphicview.cpp]: Up and down camera movement is incorrect in Thyme w3dview compared to stock w3dview, the above-mentioned change should fix that.

File: src/tools/w3dviewer/graphicview.cpp (hunk 1 above)
Comment (xezon) [reply src/tools/w3dviewer/graphicview.cpp]: Ok. So the intent was to have inverted movement on both axis. Ok then I will test your proposal.

File: src/tools/w3dviewer/graphicview.cpp (hunk 2 above)
Comment (xezon) [reply src/tools/w3dviewer/graphicview.cpp]: Fixed.

File: src/tools/w3dviewer/graphicview.cpp (hunk 1 above)
Comment (xezon) [reply src/tools/w3dviewer/graphicview.cpp]: Tested and worked. Both axis are inverted now.

**Review by jonwil — APPROVED** [review]
Looks good to me, approved.

**Pull request merged**